### PR TITLE
Component System: Update documentation (README + inline docs)

### PR DESCRIPTION
**Pull request by ItsJonQ**
This update adjusts the README + inline documentation for the new components in `@wordpress/components/ui`.

Notable changes include:

* Adding inline code examples for main component exports
* Improve Lorem Ipsum filler text
* Remove code examples that are (currently) harder to maintain (usually in `types.ts`)

## Checklist:
- [n/a] My code is tested.
- [n/a] My code follows the WordPress code style. 
- [n/a] My code follows the accessibility standards. 
- [n/a] I've tested my changes with keyboard and screen readers. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [n/a] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/components/src/ui/h-stack/README.md [hunk 1]
```diff
@@ -80,21 +80,21 @@ Determines if children should wrap.
 
 When a `Spacer` is used within an `HStack`, the `Spacer` adaptively expands to take up the remaining space.
 
-```jsx live
-import { HStack, Spacer, Text, View } from '@wp-g2/components';
+```jsx
+import { HStack, Spacer, Text, View } from '@wordpress/components/ui';
 import { ui } from '@wp-g2/styles';
```
Comment (sarayourfriend) [inline packages/components/src/ui/h-stack/README.md:85]:
```suggestion

```

File: packages/components/src/ui/h-stack/README.md [hunk 2]
```diff
@@ -103,25 +103,24 @@ function Example() {
 
 `Spacer` also be used in-between items to push them apart.
 
-```jsx live
-import { HStack, Spacer, Text, View } from '@wp-g2/components';
+```jsx
+import { HStack, Spacer, Text, View } from '@wordpress/components/ui';
 import { ui } from '@wp-g2/styles';
```
Comment (sarayourfriend) [inline packages/components/src/ui/h-stack/README.md:108]:
```suggestion

```

File: packages/components/src/ui/v-stack/README.md [hunk 3]
```diff
@@ -6,21 +6,20 @@
 
 `VStack` can render anything inside.
 
-```jsx live
-import { VStack, Text, View } from '@wp-g2/components';
-import { ui } from '@wp-g2/styles';
+```jsx
+import { VStack, Text, View } from '@wp-g2/components/ui';
```
Comment (sarayourfriend) [inline packages/components/src/ui/v-stack/README.md:10]:
```suggestion
import { VStack, Text, View } from '@wordpress/components/ui';
```

File: packages/components/src/ui/v-stack/README.md [hunk 4]
```diff
@@ -86,14 +85,14 @@ When a `Spacer` is used within an `VStack`, the `Spacer` adaptively expands to t
 ```jsx
 function Example() {
 	return (
-		<VStack css={[ui.frame.height(200)]}>
-			<View css={[ui.background.blue]}>
+		<VStack css={ [ ui.frame.height( 200 ) ] }>
+			<View css={ [ ui.background.blue ] }>
```
Comment (sarayourfriend) [inline packages/components/src/ui/v-stack/README.md:89]:
```suggestion
		<VStack>
			<View>
```

File: packages/components/src/ui/v-stack/README.md [hunk 5]
```diff
@@ -86,14 +85,14 @@ When a `Spacer` is used within an `VStack`, the `Spacer` adaptively expands to t
...
 				<Text>Ana</Text>
```
Comment (sarayourfriend) [inline packages/components/src/ui/v-stack/README.md:90]:
```suggestion
				<Text>Code</Text>
```

File: packages/components/src/ui/v-stack/README.md [hunk 6]
```diff
@@ -86,14 +85,14 @@ When a `Spacer` is used within an `VStack`, the `Spacer` adaptively expands to t
...
 			</View>
-			<Spacer css={[ui.background.blue]}>
+			<Spacer css={ [ ui.background.blue ] }>
```
Comment (sarayourfriend) [inline packages/components/src/ui/v-stack/README.md:92]:
```suggestion
			<Spacer>
```

File: packages/components/src/ui/v-stack/README.md [hunk 7]
```diff
@@ -86,14 +85,14 @@ When a `Spacer` is used within an `VStack`, the `Spacer` adaptively expands to t
...
 				<Text>Elsa</Text>
```
Comment (sarayourfriend) [inline packages/components/src/ui/v-stack/README.md:93]:
```suggestion
				<Text>is</Text>
```

File: packages/components/src/ui/v-stack/README.md [hunk 8]
```diff
@@ -86,14 +85,14 @@ When a `Spacer` is used within an `VStack`, the `Spacer` adaptively expands to t
...
 			</Spacer>
-			<View css={[ui.background.blue]}>
+			<View css={ [ ui.background.blue ] }>
```
Comment (sarayourfriend) [inline packages/components/src/ui/v-stack/README.md:95]:
```suggestion
			<View>
```

File: packages/components/src/ui/v-stack/README.md [hunk 9]
```diff
@@ -86,14 +85,14 @@ When a `Spacer` is used within an `VStack`, the `Spacer` adaptively expands to t
...
 				<Text>Olaf</Text>
```
Comment (sarayourfriend) [inline packages/components/src/ui/v-stack/README.md:96]:
```suggestion
				<Text>Poetry</Text>
```

File: packages/components/src/ui/v-stack/README.md [hunk 10]
```diff
@@ -103,18 +102,18 @@ function Example() {
...
 				<Text>Ana</Text>
 			</View>
 			<Spacer />
-			<View css={[ui.background.blue]}>
+			<View css={ [ ui.background.blue ] }>
 				<Text>Elsa</Text>
 			</View>
-			<View css={[ui.background.blue]}>
+			<View css={ [ ui.background.blue ] }>
 				<Text>Olaf</Text>
```
Comment (sarayourfriend) [inline packages/components/src/ui/v-stack/README.md:117]: Same as above

File: packages/components/src/ui/v-stack/component.js [hunk 11]
```diff
@@ -11,20 +11,22 @@ import { useVStack } from './hook';
  *
  * @example
  * ```jsx
+ * import { VStack, Text, View } from `@wp-g2/components/ui`;
```
Comment (sarayourfriend) [inline packages/components/src/ui/v-stack/component.js:14]:
```suggestion
 * import { VStack, Text, View } from `@wordpress/components/ui`;
```

**Comment by ItsJonQ** [comment]
@sarayourfriend Thanks for the fixes! It's so weird... I swear I updated them 🙈 . Maybe something happened when I saved/committed? Either way, thank you! 🙏

**Comment by ItsJonQ** [comment]
@sarayourfriend Are we okay to merge :D

**Review by sarayourfriend — APPROVED** [review]
LGTM

If we missed anything we can continue to iterate on this.

**Comment by ItsJonQ** [comment]
@sarayourfriend Amazing! Thank you 🙏 . Merging it up

**Pull request merged**